prow/cluster/jobs/rh-ibm-synergy/multicloud-operators-cluster-controller/rh-ibm-synergy.multicloud-operators-cluster-controller.master.yaml

```diff
@@ -139,7 +139,7 @@ postsubmits:
   #             key: codecov-token
   #       securityContext:
   #         privileged: true
-  - name: build-multicloud-operators-cluster-controller_postsubmit
+  - name: images-multicloud-operators-cluster-controller_postsubmit
     branches:
     - ^master$
     decorate: true
```

Comment:
Also need to add another fields 

```
    labels:
      preset-service-account: "true"
```

so that you can access secret with `kubectl` in prowjob.